website/redirects.js

```diff
@@ -1262,4 +1262,41 @@ module.exports = [
     destination: '/:splat',
     permanent: true,
   },
+  // Extracted content into `/plugins` & `/tools`
+  {
+    source: '/docs/drivers',
+    destination: '/plugins/drivers',
+    permanent: true,
+  },
+  {
+    source: '/docs/drivers/:splat*',
+    destination: '/plugins/drivers/:splat*',
+    permanent: true,
+  },
```

Comment:
I think this rule is breaking built-in drivers docs page, for example:
https://nomad-git-kevin-extract-plugins-hashicorp.vercel.app/docs/drivers/docker

We may need to list the ones that moved to the new section individually 🤔

Reply:
Ah, yea I see there are a few broken ones due to this rule

Broken:
```
docs/drivers/docker
docs/drivers/exec
docs/drivers (index)
docs/drivers/java
docs/drivers/qemu
docs/drivers/raw_exec
```

Reply:
Should be addressed in https://github.com/hashicorp/nomad/pull/11584/commits/9fe6f5d9812b4a0e384303b78140921cf777322d

Will verify when the preview updates